### ISSUE TITLE
Can this code run under Python3.6 & Pytorch 0.4.0?

### ISSUE THREAD

**Issue by lzhaozi** [huanghoujing/AlignedReID-Re-Production-Pytorch#29]
RT.
My experiment environment is Python3.6 and Pytorch0.4.0.
Thanks.

**Comment by huanghoujing** [comment]
No, it couldn't. There is some minor difference between python 2 and 3. For pytorch, version 0.4 has drastic changes. I only tried python 2.7 and pytorch 0.3. I recommend that you create a python virtual environment with these versions of packages.

Thanks for your attention.

**Comment by hsqaihkl** [comment]
I run it in win10 anaconda5.2.0,python3.6.5,pytorch0.4
@lzhaozi you should modify the error.

**Comment by taozhuang123** [comment]
@hsqaihkl 
I met some problems in win10, python3.6, pytorch0.4. Could you share your code

**Comment by taozhuang123** [comment]
@huanghoujing 
if i change to python2.7, i use windowns ,the pytorch didn't support.i hava a little sad

**Comment by ZZH2950228** [comment]
can you share your cuda and cudnn ?
